### DISCUSSION TITLE
[Bug] dist bundle contains Ant design css class overrides

### DISCUSSION THREAD

**Discussion by logovaser** [zoom/meetingsdk-web#39] [Q&A]
`zoom-meeting-2.9.7.min.js` and `zoomus-websdk.umd.min.jsz` bundles contain GLOBAL Ant design css class overrides (`ant-message`, `anticon` etc). Please remove them because they override default Antd styling for the whole app and mess up the whole app styling as a result of it.
It also has a global CSS styling for the whole `html`, `body`:
<img width="224" alt="image" src="https://user-images.githubusercontent.com/11948273/221621997-e5f8f460-9363-445f-8c59-3f59e780f271.png">

**Comment by alexleone** [comment]
This is causing issues in our application as well, please remove the global overrides
